services/director-v2/tests/unit/test_modules_dynamic_sidecar_docker_service_specs_sidecar.py

```diff
@@ -77,6 +77,7 @@ def test_dynamic_sidecar_env_vars(
         scheduler_data_from_http_request,
         app_settings,
         allow_internet_access=False,
+        metrics_collection_allowed=True,
```

Comment:
where do you actually test the the effect of setting that parameter to True or False?

Reply:
Tests were already here, for this case. See 
https://github.com/ITISFoundation/osparc-simcore/blob/bd254f58b230ff5080093653f8700fbf0f4f0177/services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py#L157-L159

Reply:
ok... not very explicit.. but ok